pkg/config/config.go

```diff
@@ -34,7 +34,7 @@ const (
 	StreamTrackerTypePacket StreamTrackerType = "packet"
 	StreamTrackerTypeFrame  StreamTrackerType = "frame"
 
-	StatsUpdateInterval = time.Second * 10
+	StatsUpdateInterval = time.Second * 30
```

Comment:
Two things @shishirng 
- This affects `NodeStats` average computation interval. I have not checked the side effects of that, but I am thinking we should probably not change that. Maybe make a different config for that.
- Are we okay to get max value stats (like RTT, jitter) once every 30 seconds?

Reply:
Thanks @boks1971 
1. Will create a new config for flushing interval for telemetry
2. We want to reduce the amount data generated for track stats. Since we consider MAX of rtt/jitter in the window, we should be fine